OpenRA.Mods.Common/Traits/Render/WithRangeCircle.cs

```diff
@@ -28,6 +28,15 @@ class WithRangeCircleInfo : ConditionalTraitInfo, IPlaceBuildingDecorationInfo
 		[Desc("Color of the circle")]
 		public readonly Color Color = Color.FromArgb(128, Color.White);
 
+		[Desc("Contrast width.")]
```

Comment:
contrast &rarr; border, and ideally update the wording of `Color` and `BorderColor` to match the other traits.